engine/api/workflowtemplate/dao.go

```diff
@@ -0,0 +1,121 @@
+package workflowtemplate
+
+import (
+	"database/sql"
+	"fmt"
+
+	"github.com/go-gorp/gorp"
+
+	"github.com/ovh/cds/engine/api/database"
```

Comment:
i'm afraid that importing the database package can be a problem i you write tests with the test package

Reply:
and often we write quite simple unit tests on all DAO functions
i hope we may have each package with coverage > 25%

Reply:
done